### PR TITLE
Update copyright

### PR DESCRIPTION
**Pull request by PastaPastaPasta**

### CODE REVIEW

**Review by xdustinface — APPROVED** [review]
utACK

File: src/batchedlogger.cpp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2018-2019 The Dash Core developers
+// Copyright (c) 2018-2020 The Dash Core developers
```
Comment (thephez) [inline src/batchedlogger.cpp:1]: Why do some files (e.g. this one) use 2020 while others use 2021? Is it based on when the file was last modified?

Comment (UdjinM6) [reply src/batchedlogger.cpp:1]: https://github.com/dashpay/dash/blob/master/contrib/devtools/copyright_header.py#L415

**Review by UdjinM6 — APPROVED** [review]
utACK

(reproduced the same diff)

**Pull request merged**